i18n/fr/docusaurus-plugin-content-docs/version-23.10/installation/installation-of-centreon-ha/installation-4-nodes.md

```diff
@@ -1073,6 +1323,24 @@ pcs resource create "ms_mysql" \
     test_table='centreon.host'
 ```
 
+</TabItem><TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
+
+```bash
+pcs resource create "ms_mysql" \
+    ocf:heartbeat:mariadb-centreon \
+    config="/etc/my.cnf.d/mariadb-server.cnf" \
+    pid="/var/lib/mysql/mysql.pid" \
+    datadir="/var/lib/mysql" \
+    socket="/var/lib/mysql/mysql.sock" \
+    binary="/usr/bin/mysqld_safe" \
+    node_list="@CENTRAL_MASTER_NAME@ @CENTRAL_SLAVE_NAME@" \
```

Comment:
```suggestion
    node_list="@DATABASE_MASTER_NAME@ @DATABASE_SLAVE_NAME@" \
```

Reply:
Cf MON-32722 and please update it in all other occurrences of node_list definition

Reply:
this command is valid for 2 nodes clusters only (where mysql is running on the centrals)